data_nutch_data_nci/conf/nutch-site.xml

```diff
@@ -63,19 +63,23 @@
 
 	<property>
 		<name>http.timeout</name>
-		<value>10000</value>
+		<value>20000</value>
 		<description>NCI OVERRIDE: Upped to 20sec from 10sec to allow time for www.cancer.gov to generate its sitemap.xml file. 20151113 - it seems 20sec is not long enough, so reverting to 10sec</description>
```

Comment:
Here the comment says it's reverted to 10 seconds, but the change moves it to 20.

Reply:
updated the comment to be 20000